tracer/src/Datadog.Trace/Debugger/Instrumentation/AsyncMethodDebuggerInvoker.cs

```diff
@@ -304,6 +316,13 @@ private static void EndMethodLogLocals(ref AsyncMethodDebuggerState asyncState)
             for (var index = 0; index < kickOffMethodLocalsValues.Length; index++)
             {
                 ref var local = ref kickOffMethodLocalsValues[index];
+                if (local == default ||
```

Comment:
See previous note about extracting this predicate into a method

Reply:
Fixed in d93f31f39ab9606ae745e1c8547fc5e7e0a6b757